resources/views/MyAccount/Components/OrderHistoryList.twig

```diff
@@ -0,0 +1,46 @@
+{{ component( "Ceres::MyAccount.Components.ChangePaymentMethod" ) }}
+{{ component( "Ceres::MyAccount.Components.OrderHistoryListItem" ) }}
+
+<script type="x/template" id="vue-order-history-list">
+    <div class="order-history-list">
+        <h4>{{ trans("Ceres::Template.orderHistory") }}</h4>
+        <p v-if="hintText" v-html="hintText">${ hintText | raw }}</p>
```

Comment:
Bitte den Text in ein div packen. Texte aus dem Editor sind selbst `<p>`-Tags, sodass sich hier die Paddings/Margins der Absätze addieren

Reply:
done